hermes-management/src/test/groovy/pl/allegro/tech/hermes/management/infrastructure/kafka/service/KafkaConsumerGroupManagerSpec.groovy

```diff
@@ -151,6 +157,68 @@ class KafkaConsumerGroupManagerSpec extends Specification {
         kafkaContainer.dockerClient.unpauseContainerCmd(containerId).exec()
     }
 
+    def "Should delete specified consumer group and retain others"() {
+        given:
+        Topic topic = createAvroTopic("pl.allegro.test.DeletionOfConsumerGroup")
+        String kafkaTopicName = kafkaNamesMapper.toKafkaTopics(topic).primary.name().asString()
+        createTopicInKafka(kafkaTopicName, 3)
+
+        and:
+        Subscription subscription = createTestSubscription(topic, "test-subscription")
+        ConsumerGroupId consumerGroupId = kafkaNamesMapper.toConsumerGroupId(subscription.qualifiedName)
+        consumerGroupManager.createConsumerGroup(topic, subscription)
+
+        and:
+        Subscription subscriptionToDelete = createTestSubscription(topic, "test-subscription-to-delete")
+        consumerGroupManager.createConsumerGroup(topic, subscriptionToDelete)
+
```

Comment:
perhaps add an assertion here to make sure that there were 2 groups before removal?